rts/Rendering/Fonts/CFontTexture.cpp

```diff
@@ -531,16 +531,16 @@ void CFontTexture::LoadBlock(char32_t start, char32_t end)
 	do {
 		alreadyCheckedFonts.insert(f);
 
-		for (auto it = map.begin(); !map.empty() && it != map.end(); ) {
-			FT_UInt index = FT_Get_Char_Index(*f, *it);
+		for (std::size_t idx = 0; idx < map.size() && !map.empty(); /*nop*/) {
```

Comment:
No idea why `!map.empty()` was present before, but `idx < map.size()` should be a sufficient condition.